sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3382,11 +3382,9 @@ object GpuOverrides extends Logging {
         override def tagExprForGpu(): Unit =
           a.schema match {
             case MapType(_: StringType, _: StringType, _) => ()
-            case MapType(kt, vt, _) => {
-              willNotWorkOnGpu("JsonToStructs only supports MapType<StringType, StringType> for " +
-                               s"input MapType schema, but received MapType<$kt, $vt>")
-            }
-            case _ => ()
+            case _ =>
+              willNotWorkOnGpu("JsonToStructs only supports MapType<StringType, StringType> " +
```

Comment:
This is not introduced by this PR but may be worth fixing

1. it' should be GpuJsonToStructs since CPU JsonToStructs very well supports it
1. The end user does not necessarily know or care about the backing Scala class

Should we say "from_json on GPU only supports MapType<StringType, StringType> ..." ?

I would also include the actual type provided by the user in that message

Reply:
Thank you! I updated the message.
For the actual type, I didn't include it because it would have been included already in the `from_json` expression, like `!Expression <JsonToStructs> from_json(StructField(a,StringType,true), StructField(b,IntegerType,true), a#116, Some(UTC)) cannot run on GPU because JsonToStructs only supports MapType<StringType, StringType> input schema, but received StructType(StructField(a,StringType,true),StructField(b,IntegerType,true))`

Reply:
good to know that input types are already included. agreed no reason to repeat it then.